src/components/Fieldset/Fieldset.jsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames/bind';
+import styles from './Fieldset.scss';
+const cx = classnames.bind( styles );
+
+const Fieldset = ( {
+	autoComplete,
+	defaultValue,
+	describedBy,
+	disabled,
+	isMultiline,
+	inputRef,
+	id,
+	label,
+	maxLength,
+	placeholder,
+	onBlur,
+	onChange,
+	onClick,
+	onFocus,
+	onInvalid,
+	pattern,
+	readOnly,
+	required,
+	type,
+} ) => {
+	// ClassNames used by both <input> and <textarea>.
+	const sharedClassNames = {
+		disabled,
+		readOnly,
+		[type]: type,
+	};
+
+	const requiredField = label && required;
+
+	return (
+		<fieldset className={styles.container}>
+			<label className={`${styles.label} ${styles.requiredLabel}`}>
+				{label}
+				{requiredField && <span className={styles.requiredAsterisk}> * </span>}
+				{isMultiline &&
+					<textarea
+						className={cx( 'textarea', { ...sharedClassNames } )}
+						rows={6}
+						aria-describedby={describedBy}
+						autocomplete={autoComplete}
+						defaultValue={defaultValue}
+						describedBy={describedBy}
+						disabled={disabled}
+						id={id}
+						maxLength={maxLength}
+						placeholder={placeholder}
+						onBlur={onBlur}
+						onChange={onChange}
+						onClick={onClick}
+						onFocus={onFocus}
+						onInvalid={onInvalid}
+						readOnly={readOnly}
+						ref={inputRef}
+						required={required}
+					/>
+				}
+
+				{!isMultiline &&
+					<input
+						className={cx( 'input', { ...sharedClassNames } )}
+						type={type}
+						aria-describedby={describedBy}
+						autocomplete={autoComplete}
+						defaultValue={defaultValue}
+						describedBy={describedBy}
+						disabled={disabled}
+						id={id}
+						maxLength={maxLength}
+						placeholder={placeholder}
+						readOnly={readOnly}
+						onBlur={onBlur}
+						onChange={onChange}
+						onClick={onClick}
+						onFocus={onFocus}
+						onInvalid={onInvalid}
+						pattern={pattern}
+						readOnly={readOnly}
+						ref={inputRef}
+						required={required}
+						type={type}
+					/>
+				}
+			</label>
+		</fieldset>
+	);
+};
+
+Fieldset.propTypes = {
+	/**
+	 * Input tag prop; boolean to accept autoCompletion or not.
+	 */
+	autoComplete: PropTypes.string,
```

Comment:
- This doesn't help me understand what this prop does. Since it's passed verbatim into the HTML attribute, let's just borrow the description from [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea).
- Don't need to mention the type in comment, this is pulled into docs from the PropType itself
- Use `PropTypes.bool.isRequired`, provide a default

```suggestion
	/**
	 * Whether the value of the control can be automatically completed by the browser..
	 */
	autoComplete: PropTypes.bool.isRequired,
```